### ISSUE TITLE
App Crashes, Bugs and Feature Requests

### ISSUE THREAD

**Issue by nr9992** [blockheaddd/BHMusic#1]
The app runs on the emulator but the logcat goes all haywire while the app starts, but when being run on an actual device, Moto G 2013, the app crashes and doesn't start. Not able to figure out why. Please look into it. Also, the album art downloaded online shows in the artist detail view but not on the album grid fragment, nor the now playing screen. The now playing queue on the now playing activity only shows the songs of the current album playing and not all the songs in the queue. Would also love to have a playlist feature too, and some more like recently played/added much like Google Play Music.

Nice work so far though. Kudos

**Comment by gusrsilva** [comment]
Hey thanks for the feedback. Yea I'm actually still working on it its in
early development but I'm releasing an alpha version soon once I get things
stable.

**Comment by nr9992** [comment]
Looking forward to the alpha version soon. Once you get a stable version, I would love to contribute towards the UX of the app

**Comment by gusrsilva** [comment]
Yea sounds good. And also was it crashing on the most recent version or
have you updated the project lately?

**Comment by nr9992** [comment]
Yes it was. I downloaded the latest build just this morning. The app
doesn't even start on my device. It crashes everytime I try opening it
- Neel Raj

**Comment by gusrsilva** [comment]
Ok thanks ill look into it.

**Comment by nr9992** [comment]
Even with the latest fix on color generator, the app still crashes everytime I run it on an actual device. I've tested it on a couple of devices and the same happens everywhere

**Comment by gusrsilva** [comment]
I tested it on a couple devices and it doesn't crash on open. Maybe
something's wrong with the project try this apk if you want
https://www.dropbox.com/s/jwk0aangamorv8d/app-release.apk?dl=0

**Comment by nr9992** [comment]
I tried the apk. Same result. Crashes on launch everytime. Don't know what is wrong
